.github/workflows/release.yml

```diff
@@ -341,7 +341,7 @@ jobs:
           git restore packages/@biomejs/wasm-nodejs/package.json
           git restore packages/@biomejs/wasm-web/package.json
 
-      - name: Copy binaries
+      - name: Generate packages
         run: node packages/@biomejs/biome/scripts/copy-binaries.mjs
```

Comment:
I think we should update the `bin` field in this script or it will be overriden by the script: https://github.com/biomejs/biome/blob/ec7c63df520103b5d8ea0090c59486574e7370dd/packages/%40biomejs/biome/scripts/copy-binaries.mjs#L26-L45